scylla/src/transport/session.rs

```diff
@@ -66,6 +66,42 @@ use crate::authentication::AuthenticatorProvider;
 #[cfg(feature = "ssl")]
 use openssl::ssl::SslContext;
 
+/// Stores default (somehow recommended) configuration options for Session.
+pub mod defaults {
+    use scylla_cql::frame::types::SerialConsistency;
+    use scylla_cql::Consistency;
+    use std::sync::Arc;
+    use std::time::Duration;
+
+    use crate::load_balancing::{LoadBalancingPolicy, RoundRobinPolicy, TokenAwarePolicy};
+    use crate::retry_policy::{DefaultRetryPolicy, RetryPolicy};
+    use crate::speculative_execution::SpeculativeExecutionPolicy;
+
+    pub fn consistency() -> Consistency {
+        Consistency::LocalQuorum
+    }
+
+    pub fn serial_consistency() -> Option<SerialConsistency> {
+        None
+    }
+
+    pub fn request_timeout() -> Option<Duration> {
+        Some(Duration::from_secs(30))
+    }
+
+    pub fn load_balancing_policy() -> Arc<dyn LoadBalancingPolicy> {
+        Arc::new(TokenAwarePolicy::new(Box::new(RoundRobinPolicy::new())))
+    }
+
+    pub fn retry_policy() -> Box<dyn RetryPolicy> {
+        Box::new(DefaultRetryPolicy::new())
+    }
+
+    pub fn speculative_execution_policy() -> Option<Arc<dyn SpeculativeExecutionPolicy>> {
+        None
+    }
+}
```

Comment:
I'm not entirely sure this should be exposed or that we should have this module at all. I checked out the PR's branch and it looks like it is only used from the execution profiles module, so I think they should just be specified there as the execution profiles become the main mechanism for specifying these parameters.

If this module helps with the refactoring then it's fine to keep it in the intermediate commits, but IMO should be removed at the end.

Reply:
@cvybhu @havaker WDYT about putting all important defaults into one place and using them through getters only?

Reply:
@cvybhu @havaker ping once more

Reply:
Ok, can I move this module as a whole to `execution_profile.rs` ? @piodul